src/processes/sort.js

```diff
@@ -0,0 +1,13 @@
+const Process = require('../processgraph/process');
+const Commons = require('../processgraph/commons');
+
+// TODO: complete implementation
+module.exports = class sort extends Process {
+
+    async execute(node, context) {
```

Comment:
Does this sort works as promised by the schema? Does it allow dates/times/strings and all that? Does it work on images or arrays? Is this tested?

Reply:
No, I will remove it for the time being until we have a solution how we can properly work on arrays.